.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  pull_request: ~
+
+env:
+  COMPOSER_ROOT_VERSION: 1.0.x-dev
+jobs:
+  tests:
+    name: Tests
+    runs-on: "ubuntu-20.04"
+    timeout-minutes: 10
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - '7.1'
+          - '7.2'
+          - '7.3'
+        composer_options: [ "" ]
+        experimental: [false]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP Action
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          extensions: redis
```

Comment:
Why is this different than the standard value in the template (`extensions: pdo_sqlite, gd`),  what difference does it make?